backend/sublet/models.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+
+User = get_user_model()
+
+
+class Offer(models.Model):
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["user", "sublet"], name="unique_offer")]
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made")
+    sublet = models.ForeignKey("Sublet", on_delete=models.CASCADE, related_name="offers")
+    # TODO: Make sure phone_number is being validated by serializers/frontend
+    email = models.EmailField(max_length=255, null=True, blank=True)
+    phone_number = models.CharField(max_length=20, null=True, blank=True)
```

Comment:
I think we would need to validate the phone number somehow, please make sure to either make this the PhoneNumberField or check in serializer update and create functions